### ISSUE TITLE
Geo::Distance 0.21+0.22 breaks Geo::Distance::XS

### ISSUE THREAD

**Issue by eserte** [bluefeet/Geo-Distance#16]
New Geo::Distance versions break the Geo::Distance::XS test suite (@gray: FYI):
```
#   Failed test 'PP version loads XS'
#   at t/02_new_pp.t line 11.
#          got: ''
#     expected: '1'
# Looks like you failed 1 test of 1.
t/02_new_pp.t ...... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

Statistical analysis (means: mostly fails with Geo::Distance 0.22):
```
****************************************************************
Regression 'mod:Geo::Distance'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.9960	      0.0028	 353.55
[1='eq_0.22']  	     -0.9960	      0.0316	 -31.50

R^2= 0.664, N= 504, K= 2
****************************************************************
```

Probably it fails also with Geo::Distance 0.21, but I did not verify this.

**Comment by eserte** [comment]
I have a new statistical analysis available. It fails also with 0.21:
```
****************************************************************
Regression 'mod:Geo::Distance'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	746723021144906112.00
[1='eq_0.21']  	     -1.0000	      0.0000	-104664800378478384.00
[2='eq_0.22']  	     -1.0000	      0.0000	-123357305907189984.00

R^2= 1.000, N= 523, K= 3
****************************************************************
```

**Comment by bluefeet** [comment]
The fix for #15 should resolve this.

**Comment by bluefeet** [comment]
Well, part of this should be fixed by #15... this ticket also refers to the independent issue that `t/02_new_pp.t` is failing in gray/geo-distance-xs.  I'll leave this ticket open for the fix to that.

**Comment by bluefeet** [comment]
Moving this to gray/geo-distance-xs#2.
